src/Patcher/MapPatcher.php

```diff
@@ -71,7 +68,14 @@ public function patch( array $base, Diff $diff ) {
 		return $base;
 	}
 
-	private function applyOperation( &$base, $key, $diffOp ) {
+	/**
+	 * @param array &$base
+	 * @param string $key
```

Comment:
Documented as `int|string` in https://github.com/wmde/Diff/blob/master/src/DiffOp/Diff/Diff.php#L409. Will fix.